x-pack/packages/ml/trained_models_utils/src/constants/trained_models.ts

```diff
@@ -57,7 +58,48 @@ export const ELASTIC_MODEL_DEFINITIONS = {
       defaultMessage: 'Elastic Learned Sparse EncodeR v1 (Tech Preview)',
     }),
   },
-} as const;
+  '.elser_model_2': {
+    version: 2,
+    default: true,
+    config: {
+      input: {
+        field_names: ['text_field'],
+      },
+    },
+    description: i18n.translate('xpack.ml.trainedModels.modelsList.elserV2Description', {
+      defaultMessage: 'Elastic Learned Sparse EncodeR v2 (Tech Preview)',
+    }),
+  },
+  '.elser_model_2_linux-x86_64': {
```

Comment:
As above, the code was right for the long term but temporarily we can use the SNAPSHOT version.

```suggestion
  '.elser_model_2_linux-x86_64_SNAPSHOT': {
```

Reply:
Added in https://github.com/elastic/kibana/pull/167000/commits/fc4d83b14b00d3e547d3017fbb5ee1b1477ce808